articles/faq/new-pricing-increases-costs

```diff
@@ -0,0 +1,6 @@
+---
+title: "**The new pricing increases our costs, can you help us avoid having to change vendors?**"
```

Comment:
You don't need the `**` around the title - it's automatically turned into a heading